### DISCUSSION TITLE
Finetuning gracetime

### DISCUSSION THREAD

**Discussion by PhXcell** [kuhn-ruess/cmdbsyncer#31] [Q&A]
Is there a way to finetune the gracetime for dorment Hosts in the syncer?

**Comment by Bastian-Kuhn** [comment]
> Is there a way to finetune the gracetime for dorment Hosts in the syncer?

Currently only Full Days can be set, and newer Version can Filter by Account which Objects will be deleted. Best create an Account of Type Maintanace, Save and you see the options. Use the Account then as Param for the command.
